components/language_filter/fondant_component.yaml

```
@@ -0,0 +1,14 @@
+name: Language filter
+description: A component that filter a provided dataframe based on the language.
+image: ghcr.io/ml6team/language_filter:latest
+
+consumes:
+  passages:
+    fields:
+      text:
+        type: string
```

```suggestion
  text:
    fields:
      data:
        type: string
```
In order to make the component reusable across different pipelines, we could use these names